Documentation changes for SEO

oxelson opened the pull request:
Changes to IDV .html, .css, and .tcl files in the userguide, workshop, and build directories.  Modifed HTML for SEO improvements.

julienchastang commented:
Jen, many thanks for putting this together. One thing I am having trouble with is there are many indentation changes here, in addition to the more substantive changes you have made, so I am having trouble separating the wheat from the chaff.

oxelson commented:
Hi Julien,

Here is a list of things I've changed (any replacements have been made in
the HTML template pages, css files, the tcl scripts (where applicable), and
any docs/*.html page that contained them):


   - Replaced:

<div class="pagetitle"></div>   with <h1></h1>
<div class="pagesubtitle"></div>  with  <h3></h3>
<div class="pagesubsubtitle"></div>  with  <h5></h5>


   - Removed  title=\"Style\" attribute from  <link rel="stylesheet"
   href="/ > tags (HTML validators whine about it's presence).


   - Added Unidata IDV (User Guide | Workshop) to templates to add
   meaningful content to the <title></title> tags.


   - Made changes to the templates to use HTML5, and CSS for formatting of
   navigation, etc.


Instead of trying to chase down every single change, I would recommend
looking the new version of docs to see the end result:
http://www.unidata.ucar.edu/staff/oxelson/idv/processed/

If something is off, you will be able to detect it here rather quickly.


On Wed, Oct 28, 2015 at 10:37 AM, Julien Chastang <notifications@github.com>
wrote:

> Jen, many thanks for putting this together. One thing I am having trouble
> with is there are many indentation changes here, in addition to the more
> substantive changes you have made, so I am having trouble separating the
> wheat from the chaff.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Unidata/IDV/pull/108#issuecomment-151903293>.
>
